### ISSUE TITLE
None of the images in content are showing up on the front-page

### ISSUE THREAD

**Issue by tobyontour** [decoupledCMS/decoupledcms.github.com#10]
The images appear to be hosted here: http://bergie.iki.fi/files/1e03f6a5bcbe4223f6a11e0a60db5207a8570387038_cms-monolithic-approach.png

But they 404.

**Comment by lsmith77** [comment]
yeah .. I am not really finding time to maintain the content anyway .. and looking for someone to take over the domain and do without something useful ..

**Comment by shieldo** [comment]
They'd been moved to being hosted on S3 - I've updated the URLs for them now.
